### PR TITLE
Fixes for TransactionList

### LINKED ISSUE #1195: Shifted text in the Tx history
text displaying
text
dev

Reproducible on develop branch.

### LINKED ISSUE #1211: No explanatory text if the token is not found
Wire 2021-07-09 at 14_31

### LINKED ISSUE #1228: Transaction lists with none or few items reload infinitely
recording (19)

### PR DESCRIPTION
**Pull request by imollov**
fixes #1228
fixes #1211
fixes #1195

### CODE REVIEW

**Comment by CedrikNikita** [comment]
loading

File: src/popup/router/components/TransactionList.vue [hunk 1]
```diff
@@ -21,7 +21,7 @@
       data-cy="loader"
     />
     <div
-      v-else-if="!transactions.length"
+      v-else-if="!filteredTransactions.length"
```
Comment (etharner) [inline src/popup/router/components/TransactionList.vue:24]: How can I reproduce it? I have this message on newly created account on develop version:
image

Comment (etharner) [reply src/popup/router/components/TransactionList.vue:24]: Ok, got it, it's after filtering

**Comment by imollov** [comment]
> loading

This is reproducible on develop. 

I propose to not show the loader when fetching next page for now as it's not in the designs either.

**Comment by CedrikNikita** [comment]
> 
> I propose to not show the loader when fetching next page for now as it's not in the designs either.

At this case user won't know that he scrolled to the bottom. I propose to drop last commit and merge the rest, since the issue i've mentioned was not supposed to be fixed within this PR.

File: src/popup/router/components/TransactionList.vue [hunk 2]
```diff
@@ -157,10 +158,7 @@ export default {
         this.updateTransactions(transactions);
       } finally {
         this.loading = false;
-      }
-      if (transactions.length) {
         this.page += 1;
-        this.checkLoadMore();
```
Comment (CedrikNikita) [inline src/popup/router/components/TransactionList.vue:163]: Without this check we will get in the situation like this, when the chunk contain not enought info to fill the whole height, and the user is not able to scroll to the bottom cause of that
Ivelin
or even like this:

this PR
image

develop
image

**Review by CedrikNikita — CHANGES_REQUESTED** [review]

**Comment by CedrikNikita** [comment]
I still have an infinite loading
Ivelin new

**Comment by imollov** [comment]
> I still have an infinite loading
> Ivelin new

I couldn't fix it with the current implementation of load more behavior.

https://github.com/aeternity/superhero-wallet/issues/1234

> At this case user won't know that he scrolled to the bottom. I propose to drop last commit and merge the rest, since the issue i've mentioned was not supposed to be fixed within this PR.

Agreed.

**Review by CedrikNikita — APPROVED** [review]

**Pull request merged**